web/src/definitions.ts

```diff
@@ -1,12 +1,12 @@
 const Definitions = {
-  studyName: 'Provide a study name to associate with your samples.',
+  studyName: 'The Study Information page requires a valid ‘Study Name’. We recommend the use of standardized, informative study names as described by the GOLD team (Mukherjee et. al., 2023).',
   piName: 'The Principal Investigator who led the study and/or generated the data.',
-  piEmail: 'An email address for the Principal Investigator.',
+  piEmail: 'The Principle Investigator\'s email is required for submission. The submitter does not have to be the Principle Investigator.',
   piOrcid: '<a href="https://orcid.org" target="_blank" rel="noopener noreferrer">ORCID iD</a> of the Principal Investigator.',
-  fundingSources: 'The source of funding for your study. As an example please see <a href="https://data.microbiomedata.org/details/study/nmdc:sty-11-8fb6t785" target="_blank" rel="noopener noreferrer">study\'s funding source</a>.',
-  linkOutWebpage: 'Link to the Principal Investigator\'s research lab webpage or the study webpage associated with this collection of samples. Multiple links can be provided.',
-  studyDescription: 'Provide a description of your study. This should include some general context of your research goals and study design. For examples, please see existing study landing pages on <a href="http://data.microbiomedata.org" target="_blank" rel="noopener noreferrer">the data portal</a>.',
-  studyOptionalNotes: 'Add any additional notes or comments about this study.',
+  fundingSources: 'The Funding Source of the research performed under this study should be provided and include an awarding program and the award or grant number. See an [existing study](https://data.microbiomedata.org/details/study/nmdc:sty-11-8fb6t785) for an example.',
+  linkOutWebpage: 'Optionally, institute, project, or research lab weblinks can be provided. If your research group or program has a website to share, please include the links.',
+  studyDescription: 'The `Study Description` should provide a summary that includes the context of your study, research, goals, and design. We recommend this be similar to abstract format, describing the entire study similar to how an abstract describes a single publication.',
```

Comment:
I think we generally don't quote these.

```suggestion
  studyDescription: 'The Study Description should provide a summary that includes the context of your study, research, goals, and design. We recommend this be similar to abstract format, describing the entire study similar to how an abstract describes a single publication.',
```